mysite/base/templates/base/events.html

```diff
@@ -53,36 +53,36 @@
     </div>
 
     <div class="module">
-      <a href="http://bostonpythonworkshop.com/"><img class="left-image" src="/static/images/bpw-180x180.jpg" /></a>
+      <a href="http://campus.openhatch.org/"><img class="left-image" src="/static/images/campus_IMG_6192_scaled.jpg" /></a>
       <div class="rest-of-box">
 	<div class="name-of-event">
-	  <a class="link-without-style" href="http://bostonpythonworkshop.com/">Boston Python Workshop</a>
+	  <a class='link-without-style' href="http://campus.openhatch.org/">Open Source Comes to Campus</a>
 	</div>
 	<div>
-	  We empower women and their friends to learn practical programming in a beginner-friendly environment.
-	  By integrating the workshop into existing Python user groups, we retain attendees and build gender
-	  diversity into the main community.
+	  Through weekend workshops at colleges, we teach the next generation of computer science students
+	  how to participate in open source communities. The workshop is a collaboration with existing
+	  on-campus computing groups so that open source becomes a lasting part of campus culture.
 	</div>
       </div>
       <div class="little-read-more">
-	<a href="http://bostonpythonworkshop.com/">Read more &raquo;</a>
+	<a href="http://campus.openhatch.org/">Read more &raquo;</a>
       </div>
     </div>
 
     <div class="module">
-      <a href="http://campus.openhatch.org/"><img class="left-image" src="/static/images/campus_IMG_6192_scaled.jpg" /></a>
+      <a href="http://bostonpythonworkshop.com/"><img class="left-image" src="/static/images/bpw-180x180.jpg" /></a>
       <div class="rest-of-box">
-	<div class="name-of-event">
-	  <a class='link-without-style' href="http://campus.openhatch.org/">Open Source Comes to Campus</a>
-	</div>
-	<div>
-	  Through weekend workshops at colleges, we teach the next generation of computer science students
-	  how to participate in open source communities. The workshop is a collaboration with existing
-	  on-campus computing groups so that open source becomes a lasting part of campus culture.
-	</div>
+  <div class="name-of-event">
```

Comment:
Try choosing the "indent using spaces" option at the bottom right of your sublime text editor and set the tab width to 4 spaces. I suspect your tab width is currently set to two spaces.